core/src/main/java/org/apache/iceberg/AllManifestsTable.java

```diff
@@ -43,6 +52,7 @@
  * This table may return duplicate rows.
  */
 public class AllManifestsTable extends BaseMetadataTable {
+  private static final int REF_SNAPSHOT_ID = 18;
```

Comment:
Are we keeping this a constant because we refer to it again in the isSnapshotRef function? I just wonder because we don't do this for any of our other fields in the schema.

Reply:
Yea that's it actually, its used in two places.  Thoughts?